src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.UserDto;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/user")
+@CrossOrigin("*")
+public class UserController {
```

Comment:
In this controller i can't find blockUser endpoint and endpoint returned random key. BlockUser method should change  status field in User. Maybe it could be 0-loggedOut, 1-loggedIn, 2-blocked. It's only proposition to dicussion

Reply:
Endpoint generating a random key after entering correct user data:
- I think it should accept UserDto and return some LoginDto or SessionDto (values ​​from login.spec.json)

Reply:
I think you are right. And I like your proposal about 2 means blocked. Also, because our description of this program is saying only about creating user I will delete rest of the methods except addUser. Worst case scenario I will be going with commits back and forth.